packages/web-components/src/components/filter-panel/filter-panel-modal.ts

```diff
@@ -26,21 +26,81 @@ import { carbonElement as customElement } from '../../internal/vendor/@carbon/we
 const { prefix } = settings;
 const { stablePrefix: ddsPrefix } = ddsSettings;
 
+const WITHIN = Node.DOCUMENT_POSITION_CONTAINED_BY;
+const PRECEDING =
+  Node.DOCUMENT_POSITION_PRECEDING | Node.DOCUMENT_POSITION_CONTAINS;
+const FOLLOWING =
+  Node.DOCUMENT_POSITION_FOLLOWING | Node.DOCUMENT_POSITION_CONTAINED_BY;
+
+function tryFocusElems(
```

Comment:
In the interest of keeping this DRY, could we make it a protected method in `DDSExressiveModal`? Then we would inherit it in `DDSFilterPanelModal` and have it handy for use without duplicating it.

Reply:
Sure I can take a crack at it.